x-pack/plugins/cases/public/containers/use_post_comment.tsx

```diff
@@ -84,6 +85,9 @@ export const usePostComment = (): UsePostComment => {
             );
           }
           dispatch({ type: 'FETCH_FAILURE' });
+          if (throwOnError) {
```

Comment:
Could you please add a test where you test this behavior?

Reply:
Added!